vent/gui/styles.py

```diff
@@ -18,7 +18,7 @@
 CONTROL_TEXT = BACKGROUND_COLOR
 HANDLE_HEIGHT = 10
 SLIDER_WIDTH = 80
-SLIDER_HEIGHT = 50
+SLIDER_HEIGHT = 20 #50
```

Comment:
same thing here. sorry if these dont' work as expected. the plan is to make these values adaptive to screen size so also fine to commit for now.